spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/DenseToSparse.scala

```diff
@@ -24,29 +24,31 @@ import scala.reflect.ClassTag
 
 /**
  * Convert DenseTensor to SparseTensor.
- * @param ev$1
- * @param ev
+ * @param isBackward whether update gradInput during backward
  * @tparam T The numeric type in the criterion, usually which are [[Float]] or [[Double]]
  */
-class DenseToSparse[T: ClassTag](implicit ev: TensorNumeric[T]) extends TensorModule[T] {
+class DenseToSparse[T: ClassTag](val isBackward: Boolean = true
```

Comment:
Please change `isBackward` to `val propagateBack: Boolean = true, // propagate gradient back`, like SpatialConvolution.